internal/cmd/skupper/token/kube/token_issue.go

```diff
@@ -80,7 +80,12 @@ func (cmd *CmdTokenIssue) ValidateInput(args []string) error {
 		if !ok {
 			validationErrors = append(validationErrors, fmt.Errorf("there is no active skupper site in this namespace"))
 		} else {
-			cmd.grantName = siteName + "-" + uuid.New().String()
+			ok, siteName = utils.SiteLinkAccessEnabled(siteList)
+			if !ok {
+				validationErrors = append(validationErrors, fmt.Errorf("A site must have link access enabled before a token can be created"))
```

Comment:
```suggestion
				validationErrors = append(validationErrors, fmt.Errorf("You must enable link access for this site before you can create a token."))
```
trying to avoid passive voice

Reply:
+1

Reply:
done